src/main/java/com/snowflake/kafka/connector/records/SnowflakeAvroConverter.java

```diff
@@ -41,11 +41,11 @@
 
 public class SnowflakeAvroConverter extends SnowflakeConverter
 {
-  private SchemaRegistryClient schemaRegistry = null;
+  SchemaRegistryClient schemaRegistry = null;
```

Comment:
Nit: Maybe make these protected instead of public since it should only be used by the class extending it?

Reply:
Ah, interesting.  Its been a while since I have used Java, clearly :)

Reply:
no worries! My java is a little rusty too :)